### PR TITLE
Smart dashboard

### PR DESCRIPTION
**Pull request by Math-ias**
It still needs drive train metrics in the dash but that's fine, we can always add more to this branch later. I think we should move towards getting stuff merged into master a lot quicker, so we don't have a testing branch merge that makes 50 new commits.

### CODE REVIEW

File: src/org/robockets/tshirtcannon/Robot.java [hunk 1]
```diff
@@ -38,6 +39,7 @@ public void robotInit() {
     	robotMap = new RobotMap();
 		oi = new OI();
     	buttonThread.start();
+    	(new UpdateDashboard()).start();
```
Comment (jakekinsella) [inline src/org/robockets/tshirtcannon/Robot.java]: This seems a bit iffy to me, but if you are okay with it...

File: src/org/robockets/tshirtcannon/subsystems/cannon/UpdateCannonDashboard.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import org.robockets.tshirtcannon.Robot;
+import org.robockets.tshirtcannon.RobotMap;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ * Update SmartDashboard with information regarding the Cannon subsystem.
+ */
+public class UpdateCannonDashboard extends Command {
```
Comment (jakekinsella) [inline src/org/robockets/tshirtcannon/subsystems/cannon/UpdateCannonDashboard.java:12]: This is pretty awesome. I might do this for ButtonManager

**Comment by Math-ias** [comment]
Boop how about now? Is this good?

**Comment by jakekinsella** [comment]
:+1:

**Pull request merged**